test/suites/nodeclaim/garbage_collection_test.go

```diff
@@ -147,3 +150,44 @@ var _ = Describe("NodeClaimGarbageCollection", func() {
 		env.EventuallyExpectNotFound(node)
 	})
 })
+
+func ExpectInstanceProfileCreated(instanceProfileName *string) {
```

Comment:
```suggestion
func ExpectInstanceProfileCreated(name string) {
```

Why does it need to be a string pointer?